docs/source/code-generation/introduction.mdx

```diff
@@ -50,7 +52,7 @@ The Apollo code generation engine can be run using one of two methods:
 
 **This is the recommended method for most use cases.**
 
-The [Codegen CLI](./codegen-cli) is the simplest way to get started with Apollo iOS. The correct version of the CLI is automatically bundled with our SPM and Cocoapod packages, so you never have to worry about installation or updates.
+The [Codegen CLI](./codegen-cli) is the simplest way to get started with Apollo iOS. The correct version of the CLI is automatically bundled with our Cocoapod package, so you never have to worry about installation or updates.
```

Comment:
Why the change to remove SPM here?

Reply:
This may have been my change when I pulled out the plugins and just wasn't added back when you created the new installation plugin?